src/connector/sql/sqlGeneration.ts

```diff
@@ -470,11 +613,7 @@ function constructSqlQuery(
 
   let orderByClause = null;
 
-  if (
-    sqlQueryCtx.orderBy != null &&
-    sqlQueryCtx.orderBy != null &&
-    sqlQueryCtx.orderBy.elements.length > 0
-  ) {
+  if (sqlQueryCtx.orderBy && sqlQueryCtx.orderBy.elements.length > 0) {
```

Comment:
`elements` is a required key in the `sdk.OrderBy` type, so I think if `orderBy` exists, then `elements` must also exist. Otherwise, `sdk.OrderBy` wouldn't have been parsed correctly.